pyfar/plot/utils.py

```diff
@@ -236,47 +247,89 @@ def shortcuts(show=True):
+                           f'     - :py:func:`~pyfar.plot.{p}`\n')
+
+        sc_str += ("\nNote that not all plots are available for TimeData and "
+                   "FrequencyData objects as detailed in the documentation of "
+                   "plots.\n\n")
+
+        # shortcut for controlling the plot
+        if layout == "console":
+            sc_str += ("Use these shortcuts to control the plot\n"
+                       "---------------------------------------\n")
+        elif layout == "sphinx":
+            sc_str += "**Use these shortcuts to control the plot**\n\n"
+            sc_str += (
+                ".. list-table::\n"
+                "   :widths: 25 100\n"
+                "   :header-rows: 1\n\n"
+                "   * - Key\n"
+                "     - Action\n")
+
         ctr = short_cuts["controls"]
         for action in ctr:
             if "key_verbose" in ctr[action]:
                 key = ctr[action]["key_verbose"]
             else:
                 key = ctr[action]["key"]
-            print(f'{", ".join(key)}: {ctr[action]["info"]}')
-        print(" ")
-
-        print("Notes on plot controls")
-        print("----------------------")
-        print("Moving and zooming the x and y axes is supported by all plots.")
-        print(" ")
-        print(("Moving and zooming the colormap is only supported by plots "
-               "that have a colormap."))
-        print(" ")
-        print(f"Toggling the x-axis is supported by: {', '.join(x_toggle)}")
-        print(" ")
-        print(f"Toggling the y-axis is supported by: {', '.join(y_toggle)}")
-        print(" ")
-        print(f"Toggling the colormap is supported by: {', '.join(cm_toggle)}")
-        print(" ")
-        print(("Toggling between line and 2D plots is not supported by:"
-               " spectrogram"))
-        print(" ")
-
-    return short_cuts
+
+            if layout == "console":
+                sc_str += f'{", ".join(key)}: {ctr[action]["info"]}\n'
+            else:
+                sc_str += (f'   * - {", ".join(key)}\n'
+                           f'     - {ctr[action]["info"]}\n')
+
+        # notes on plot controls
+        if layout == "console":
+            sc_str += ("\nNotes on plot controls\n"
+                       "----------------------\n")
+        elif layout == "sphinx":
+            sc_str += "\n**Notes on plot controls**\n\n"
+
+        sc_str += ("- Moving and zooming the x and y axes is supported by all "
+                   "plots.\n"
+                   "- Moving and zooming the colormap is only supported by "
+                   "plots that have a colormap.\n"
+                   "- Toggling the x-axis is supported by: "
+                   f"{', '.join(x_toggle)}\n"
+                   "- Toggling the y-axis is supported by: "
+                   f"{', '.join(y_toggle)}\n"
+                   "- Toggling the colormap is supported by: "
+                   f"{', '.join(cm_toggle)}\n"
```

Comment:
Is it possible to link here with `:py:func:`~pyfar.plot.`?

Reply:
Yes, absolutlely